watertap/examples/flowsheets/crystallization/multi_effect_crystallizer.py

```diff
@@ -0,0 +1,306 @@
+import pandas as pd
```

Comment:
You also would need to add a test file for this file.

Reply:
Will add a test file for this, thanks!